File: app/src/main/java/ro/alexmamo/weatherapp/data/OpenWeatherMapApi.java
package ro.alexmamo.weatherapp.data;

import retrofit2.Call;
import retrofit2.http.GET;
import retrofit2.http.Query;
import ro.alexmamo.weatherapp.city.models.CurrentWeather;

public interface OpenWeatherMapApi {
    @GET("weather")
    Call<CurrentWeather> getCurrentWeather(@Query("q") String cityName, @Query("appid") String apiKey);
}

File: app/src/main/java/ro/alexmamo/weatherapp/data/OpenWeatherMapApi.kt
package ro.alexmamo.weatherapp.data

import retrofit2.Call
import retrofit2.http.GET
import retrofit2.http.Query
import ro.alexmamo.weatherapp.city.models.CurrentWeather

interface OpenWeatherMapApi {
    @GET("weather")
    fun getCurrentWeather(@Query("q") cityName: String, @Query("appid") apiKey: String): Call<CurrentWeather>
}